.github/workflows/aggregator-dockerhub.yml

```
@@ -0,0 +1,29 @@
+name: aggregator-dockerhub
+
+on:
+  push:
+    branches:
+      - 'main'
+    paths:
+      - 'aggregator/**'
+      - '.github/workflows/aggregator-dockerhub.yml'
+
+defaults:
+  run:
+    working-directory: ./aggregator
+
+env:
+  DENO_VERSION: 1.x
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: denoland/setup-deno@v1
+      with:
+        deno-version: ${{ env.DENO_VERSION }}
+    - run: git show HEAD
+    - run: echo ${{ secrets.DOCKERHUB_TOKEN }} | docker login --username blswalletghactions --password-stdin
+    - run: ./programs/build.ts --image-name blswallet/aggregator --image-only --also-tag-latest --push
```

There's little cost to pushing up every commit. This also will make it easier to share agg images early when testing bug fixes with external devs, and will make it easier to test the image in the future to ensure it is working.

```suggestion
    paths:
      - 'aggregator/**'
      - '.github/workflows/aggregator-dockerhub.yml'

defaults:
  run:
    working-directory: ./aggregator

env:
  DENO_VERSION: 1.x

jobs:
  push:
    runs-on: ubuntu-latest

    steps:
    - uses: actions/checkout@v3
    - uses: denoland/setup-deno@v1
      with:
        deno-version: ${{ env.DENO_VERSION }}
    - run: git show HEAD
    - run: echo ${{ secrets.DOCKERHUB_TOKEN }} | docker login --username blswalletghactions --password-stdin
    - run: ./programs/build.ts --image-name blswallet/aggregator --image-only --push
```

---

You can then tag in another step on `main` or git tag/GH release

```yaml
  push-latest
    runs-on: ubuntu-latest
    if: github.ref == 'refs/heads/main'
    needs: [push]

    steps:
    - run: docker tag IMAGE blswallet/aggregator:latest # or version
    - run: docker push blswallet/aggregator:latest
```